fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -40,352 +40,54 @@
 
 #include "flow/actorcompiler.h" // This must be the last #include.
```

Comment:
looks like we don't even need to split this file?

Reply:
And if we do need to split, this .h file don't need to be .actor.h

Reply:
We need the impl file because of how we're templating, but I can make this a non actor file